src/gsAssembler/gsAssembler.hpp

```diff
@@ -623,9 +623,9 @@ template<class T>
 gsField<T> gsAssembler<T>::constructSolution(const gsMatrix<T>& solVector,
                                                 int unk) const
 {
-    gsMultiPatch<T> * result = new gsMultiPatch<T>;
+    typename gsMultiPatch<T>::Ptr result(new gsMultiPatch<T>);
```

Comment:
Makes for me more sense. Pointer needed to survive scope, therefor I use a smartpointer.